salt/runc/installed.sls

```diff
@@ -0,0 +1,10 @@
+{% from "repo/map.jinja" import repo with context %}
```

Comment:
Is it 'good practice' to let one top-level state tree reference another?

Reply:
I think we may want to have every top-level 'directory' to be fully self-standing, i.e. not depend on anything else. However, that's a project-wide guideline/rule to be defined by the team, if applicable. Since there's no rule right now, can keep this as-is.

Reply:
Following our discussion on slack, I would say that a map.jinja should be referenced only by state in the same directory or in sub directories of the said map.jinja.